src/Resolver/Proxy.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\Upward\Resolver;
+
+use Magento\Upward\Definition;
+use Zend\Http\Client;
+
+class Proxy extends AbstractResolver
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getIndicator(): string
+    {
+        return 'proxy';
```

Comment:
Indicator should be `target` not `proxy`: https://github.com/magento-research/pwa-studio/tree/release/2.0/packages/upward-spec#resolver-type-inference